internal/runbits/requirements/requirements.go

```diff
@@ -81,22 +92,34 @@ func executePackageOperation(prime primeable, packageName, packageVersion string
 		language, err := model.LanguageByCommit(pj.CommitUUID())
 		if err == nil {
 			langName = language.Name
-			ns = model.NewNamespacePkgOrBundle(langName, nsType)
+			if nsType == model.NamespacePackage || nsType == model.NamespaceBundle {
```

Comment:
Should do this conditional before line 92. There's no point looking up the language unless we are looking at one of these namespaces it seems like.